### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheGamer1002**
grammar on the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,15 +50,15 @@ Works with cactus, sugar cane, saplings, hoppers, dispensers and droppers.
...
+Chickens can be sheared to get feathers. Beware! every time you shear a chicken, it gets damaged!  
 Baby chickens can't be sheared.  
 * Type: `boolean`  
 * Default value: `false`  
 * Required options: `true`, `false`  
 * Categories: `EXTRAS`, `FEATURE`  
   
 ## clericsFarmWarts
-Clerics can warm nether farts.  
+Clerics can warm nether warts.
```
Comment (DragonEggBedrockBreaking) [inline README.md:61]: This is intentional

**Review by DragonEggBedrockBreaking — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheGamer1002) [reply README.md:61]: but why? No offense, but that doesn’t seem like the best idea

Comment (DragonEggBedrockBreaking) [reply README.md:61]: Well, **warming farts** is faster than **farming warts**. @gnembon has marked multiple similar pull requests and issue reports as WAI (works as intended).

Comment (TheGamer1002) [reply README.md:61]: um... you’re not wrong, but I think we should go for the more understood option. People may be confused and also, while I’m a fan of humor, we should behave like professionals here.

Comment (DragonEggBedrockBreaking) [reply README.md:61]: > we should behave like professionals here.

This is literally a block game...

**Comment by altrisi** [comment]
Not only that, please do note that the README is automatically generated from CarpetExtraSettings, so this would be restored in next release.

(also, even if it wasn't WAI you forgot changing warm too)

File: README.md (hunk 1 above)
Comment (DragonEggBedrockBreaking) [inline README.md:61]: If this is ever going to be merged, then at least this needs to be done (as @altrisi) said.
```suggestion
Clerics can farm nether warts.
```

Comment (DragonEggBedrockBreaking) [reply README.md:61]: and it also needs to be changed here:
https://github.com/gnembon/carpet-extra/blob/5bf3ffa2b167422fda9b743dd69ecfc0ce49db25/src/main/java/carpetextra/CarpetExtraSettings.java#L106

Comment (DragonEggBedrockBreaking) [reply README.md:61]: Feel free to mark this thread as resolved, it is up to gnembon in the end.

**Comment by DragonEggBedrockBreaking** [comment]
and it also needs to be changed here:
https://github.com/gnembon/carpet-extra/blob/5bf3ffa2b167422fda9b743dd69ecfc0ce49db25/src/main/java/carpetextra/CarpetExtraSettings.java#L106

**Pull request closed without merging**